src/app/routes/utils/fetchDataFromBFF/index.ts

```diff
@@ -52,17 +52,13 @@ export default async ({
     page,
   });
 
-  const agent = isLocal ? undefined : await getAgent();
-  const timeout = isLocal && BFF_IS_LOCAL ? 60000 : null;
-  const optHeaders: OptHeaders =
-    isLocal && !BFF_IS_LOCAL
-      ? undefined
-      : {
-          'ctx-service-env': getEnvironment(pathname),
-        };
+  const agent = isLocal || BFF_IS_LOCAL ? undefined : await getAgent();
+  const timeout = isLocal || BFF_IS_LOCAL ? 60000 : null;
+  const optHeaders: OptHeaders = {
+    'ctx-service-env': environment,
+  };
 
-  if (BFF_IS_LOCAL && optHeaders) {
-    optHeaders['ctx-service-env'] = process.env.BFF_ENV || 'live';
```

Comment:
I wanted to remove the need for another enviroment variable, and ensure we can use renderer_env to determine which environment to use instead